examples/basic/README.md

```diff
@@ -0,0 +1,13 @@
+Basic example with sources and tests separated.
+
+Install
+===
+
+jspm install
+
+npm install
```

Comment:
`npm install` should come before `jspm install` in case the user doesn't have `jspm` globally installed.